### ISSUE TITLE
Layout/EnsureRescueAlignment does not work with inline access modifiers

### ISSUE THREAD

**Issue by JoeWoodward** [rubocop-hq/rubocop#7481]
## Expected behavior

When using inline access modifiers the Layout/EnsureRecueAlignment cop should align rescue with the start of line not with def.

```ruby
    private_class_method def self.decode_header(header)
      Base64.decode64(header)
    rescue StandardError
      nil
    end
```

## Actual behavior

It aligns with def.

```ruby
    private_class_method def self.decode_header(header)
      Base64.decode64(header)
                         rescue StandardError
                           nil
    end
```

## Steps to reproduce the problem

Using the defaults, the above is produceable

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ bundle exec rubocop -V
0.70.0 (using Parser 2.6.4.0, running on ruby 2.5.3 x86_64-darwin18)
```

**Comment by maxshort** [comment]
I can't reproduce this on the latest master.

It looks like it auto-corrects to the expected behavior above (aligning with `private_class_method`).

```
  private_class_method def self.decode_header(header)
    Base64.decode64(header)
  rescue StandardError
    nil
  end
```

**Comment by marcandre** [comment]
Thanks for checking @maxshort 

I'll close this, then 👍
